src/components/UploadPhoto.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+    <div class="upload-container">
+      <div class="input">
+        <input placeholder="nom du fichier" type="text" v-model="fileName">
+      </div>
+      <div class="round-button-container" >
+        <button @click="tracking" class="upload button">
+            <i class="material-icons">cloud_upload</i>
+        </button>
+      </div>             
+    </div> 
+</template>
+
+<script>
+export default {
+    name: 'UploadPhoto',
+    data () {
+        return {
+            fileName: ""
+        }
+    },
+    methods: {
+        tracking(){
+            this.$parent.fileName = this.fileName;
```

Comment:
You shouldn't use `this.$parent`. It creates a tight coupling between parent and children, vue is component oriented especially to avoid component coupling. Instead, use custom events.

[Read more about it here](https://v1.vuejs.org/guide/components.html#Parent-Child-Communication)